uqcsbot/scripts/welcome.py

```diff
@@ -17,9 +17,10 @@
     + " members well-being and safety. You can view a copy of this code of conduct at",
     "https://github.com/UQComputingSociety/code-of-conduct",
     "Your friendly committee and admins are"
-    + " Nicholas Lambourne (@ndl), Jack Caperon (@jam), Bradley Stone (@Σ),"
-    + " Joshua Sutton (@jsutton), Ella de Lore (@Ella), Matthew Lake (@mgtlake),"
-    + " Ishraque Zahin (@Ishraque), Ryan Kurz (@lighthou) and Raghav Mishra (@rm)",
+    + " Madhav Kumar (@kumar-ish), Matthew Low (@mcl), Bradley (@Σ),"
+    + " Sannidhi Bosamia (@sanni), Jack Caperon (@jam), James Copperthwaite (@jcopperthwaite),"
+    + " Kenton Lam (@Kenton), Nicholas Lambourne (@noodle), Olivia Mackenzie-Ross (@Liv),"
+    + " Raghav Mishra (@rm) and Brian Riwu Kaho (@pipInstall)"
```

Comment:
You need a comma at the end of this line; otherwise you have "...Brian Riwu Kaho ([at]pipInstall)For a list of ..."

Reply:
Also, add a colon at the end of the String in this line.